ui/app/templates/components/secret-create-or-update.hbs

```diff
@@ -116,14 +116,19 @@
     {{/if}}
 
     <div class="field is-grouped box is-fullwidth is-bottomless">
-      <div class="control">
+      <Hds::ButtonSet>
         <Hds::Button @text="Save" type="submit" disabled={{@buttonDisabled}} data-test-secret-save={{true}} />
-      </div>
-      <div class="control">
-        <SecretLink @mode="list" @secret={{@model.parentKey}} class="button">
```

Comment:
Another example of replacement https://github.com/hashicorp/vault/pull/23698/files#diff-5a1b6fbb9e7b80234fab1843f373c27c90f123c21dbf303efaa765da9563ef96L95-L101